src/index.js

```diff
@@ -5,7 +5,9 @@
 const fs = require('fs');
 const feedQueue = require('./feed-queue');
 const feedWorker = require('./feed-worker');
+const { logger } = require('./lib/logger');
 
+const log = logger.child({ module: 'basic-queue' });
```

Comment:
The naming for the child logger should be consistent with other usages. In this case, it should be `{module: 'index'}`, for `index.js`.

Reply:
Module as `index` will be pretty vague to understand in the logs, and this index.js file is something to get us started for the queue handling, that's why logs with the module name as `basic-queue` make more sense to me.